### ISSUE TITLE
Filter events by time period

### ISSUE THREAD

**Issue by danyakarras** [danyakarras/Venue#4]
Add filtering by "today", "this week", "this month" in addition to filtering by venue name.
This is something to come back to at the very end, really NOT PRIORITY

**Comment by dijana94** [comment]
yup, done. I haven't added filtering by dates yet... but I feel like that'd be a bonus feature that we can worry about later once we have basic functionality.
